packages/hdwallet-native/package.json

```diff
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "@bitcoinerlab/secp256k1": "^1.1.1",
+    "@noble/curves": "^1.4.0",
```

Comment:
note @noble/ed25519 is alsi available as a lightweight version but @noble/curves has more safety/features implemented